### PR TITLE
fix(proposal): remove redundant generation field

### PR DESCRIPTION
**Pull request by oetyng**
It was later introduced in the SAP, which is already part of the types where the generation field was previously added.

### CODE REVIEW

**Review by grumbach — APPROVED** [review]
LGTM! Great improvement!

File: sn_node/src/node/dkg/session.rs [hunk 1]
```diff
@@ -371,12 +371,9 @@ impl Session {
             secret_key_share: outcome.secret_key_share,
         };
 
-        let generation = self.session_id.membership_gen;
```
[the remark this answers is not in the document]
Comment (oetyng) [reply sn_node/src/node/dkg/session.rs:374]: Yup!

```
pub fn from_dkg_session(session_id: DkgSessionId, pk_set: PublicKeySet) -> Self {
        Self::new(
            session_id.elder_peers(),
            session_id.prefix,
            session_id
                .bootstrap_members
                .iter()
                .cloned()
                .map(|n| n.into_state()),
            pk_set,
            session_id.membership_gen, /// <-------- Here
        )
    }
```

**Review by davidrusu — APPROVED** [review]
Nice find!

**Pull request merged**